pegasus/sites.v3/code.org/public/educate/resources/videos.md

```diff
@@ -137,15 +138,15 @@ We are building a growing library of educational videos available for re-use by
 
 <div style='clear:both'></div>
 
-<div style="float:left; padding:10px; width:49%">
+<div class="col-50" style="float: left; padding: 10px;">
 
 <% facebook = {:u=>'https://youtu.be/jsUN0NV5RfQ'} %>
 <% twitter = {:url=>'https://youtu.be/jsUN0NV5RfQ', :related=>'codeorg', :text=>'Chris Bosh explains REPEAT UNTIL @codeorg'} %>
 <%=view :display_video_thumbnail, id: "repeat_until", video_code: "jsUN0NV5RfQ", caption: "Chris Bosh explains REPEAT UNTIL", play_button: 'center', facebook: facebook, twitter: twitter, download_path: "//s3.amazonaws.com/videos.code.org/2015/social/bosh-repeat-general.mp4" %>
 
 </div>
 
-<div style="float:left; padding:10px; width:49%">
+<div class="col-50" style="float: left; padding: 10px;">
```

Comment:
The number of times this gets used makes me wonder if it'd be good to add some more styles now :)